src/firefly/js/ui/tap/ObsCore.jsx

```diff
@@ -0,0 +1,982 @@
+import React,  {useState, useEffect, useReducer} from 'react';
+import PropTypes from 'prop-types';
+import Enum from 'enum';
+import {get, has, isUndefined, set} from 'lodash';
```

Comment:
there are still some unused imports (ESlint shows it): useReducer, has, isUndefined, set, getField